pygeo/mphys/mphys_dvgeo.py

```diff
@@ -218,16 +225,16 @@ def nom_addThicknessConstraints2D(self, name, leList, teList, nSpan=10, nChord=1
         else:
             self.add_output(name, distributed=True, shape=(0,))
 
-    def nom_addThicknessConstraints1D(self, name, ptList, nCon, axis):
-        self.DVCon.addThicknessConstraints1D(ptList, nCon, axis, name=name)
+    def nom_addThicknessConstraints1D(self, name, ptList, nCon, axis, scaled=True):
```

Comment:
Why is `scaled` added here? It is fine to add it, just curious about the motivation for it.

Reply:
I think the current formulation sets the scaled thickness constraint only. This will allow having absolute thickness constraint.

Reply:
Right, just wondering if there is a unique case that needs this. But, I'm totally fine with adding it regardless.

Reply:
It was added because my test case uses the option in a thickness constraint. It's an option in DVCon but no one had needed it so it wasn't added to the MPhys layer yet.

Reply:
Sounds good! The default behavior is consistent anyways so it backwards compatibly.